content/blog/2025/08/08/2025-08-08-board-officer-election-announcement.adoc

```diff
@@ -0,0 +1,123 @@
+---
+:layout: post
+:title: "Jenkins Board and Officer Elections 2025 - Nominations Open"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- markewaite
+- basil
```

Comment:
```suggestion
- slide
```
Not sure if this was intended to be left as Mark & Basil, but just noticed upon preview. Sorry for not realizing this previously!